scripts/ts-bin/multiplex_transform.ts

```diff
@@ -37,7 +37,7 @@ export class Multiplex extends stream.Transform {
   public outputStreams : { [streamName:string] : stream.Writable } = {};
   public finishedPromises : { [streamName:string] : Promise<void> } = {};
   public fullStreams : { [streamName:string] : null } = {};
-  private transformCb : ((error:Error|null) => void)[] = [];
+  private transformCb : ((error:Error|undefined) => void)[] = [];
```

Comment:
I know that you're just fixing the types here, but it would help to have a comment to explain the undefined case here.

Reply:
Looks like typescript libraries slightly changed here and this should just be Function. But I added a comment about it too (The typing comes from node stream transform library)

Reply:
Again, not seeing the comment here. But github's code review interface is not great so I may have just missed it--let me know!

Reply:
this was one committed, but you didn't notice the "outdated" thing at the top of github review pane - you have to click on it to see the latest changes sometimes.